include/soci/sqlite3/soci-sqlite3.h

```diff
@@ -332,6 +332,7 @@ struct sqlite3_session_backend : details::session_backend
         }
 
     }
+    data_type integer_type_;
```

Comment:
The clean-up in https://github.com/SOCI/soci/pull/720/commits/a975157a88254bdd3a9377b6c5395f9839bdaf0f missed this one, no?

Reply:
yes. i have missed that line. now updated with removing of that line.